DESCRIPTION

```diff
@@ -28,6 +28,7 @@ Imports:
     RJSONIO,
     scales,
     SqlRender,
+    ParallelLogger,
```

Comment:
While I don't think this will be an extra dependency for most users, this is only used in a single function call so I would suggest removing it